packages/adapter-sveltekit/src/hooks/snippet-read.ts

```diff
@@ -0,0 +1,163 @@
+import type {
+	SliceMachineHelpers,
+	SnippetReadHook,
+} from "@slicemachine/plugin-kit";
+import { stripIndent } from "common-tags";
+
+import type { PluginOptions } from "../types";
+
+const dotPath = (...segments: (string | string[])[]): string => {
+	return segments.flat().join(".");
+};
+
+const format = async (input: string, helpers: SliceMachineHelpers) => {
```

Comment:
I started to refactor the code to use a shared `format()` function, but it would only save one bit of configuration: `plugins: ["prettier-plugin-svelte"]`, which isn't needed in all cases. All other configuration is location-specific, so abstracting to a shared function doesn't save much code.

I think we should leave the code as is since it avoid unnecessary abstraction.